### ISSUE TITLE
Multiple translations

### ISSUE THREAD

**Issue by azmeuk** [dialect-app/dialect#278]
Some words, like the English word *right*, have several meanings depending on the context.

The Google Translate displays all the possible meanings for a translated word. I think this would be awesome to have such a feature in Dialect.

What do you think?
![Screenshot 2022-08-23 at 13-23-39 Google Traduction](https://user-images.githubusercontent.com/60163/186146132-f80faa8a-a803-4dc5-88de-30364fc641b8.png)

**Comment by Julvenzor** [comment]
Yes. English is a language with too many polysemantic words. Without context, any word means anything.

Regards.

**Comment by fushinari** [comment]
We're planning to implement a definition view eventually, which may include this feature too. But it's not a priority right now, especially considering development itself has slowed down.

**Comment by jeblad** [comment]
Some translation engines provide alternate translated texts. In particular deeper translations imply more available variations, contrary to shallow engines that usually pride few alternatives.

There is also the problem whereby one word in the translated text is wrong, but changes to it will imply other changes. For example changes in gender.
